misc/get-doc.js

```diff
@@ -1,6 +1,43 @@
-var doc = require('./api-doc');
+const fs = require('fs');
+const doc = require('./api-doc');
+const excluded = new Set([
+  'util'
+]);
+
+function isDirectory(path) {
+  return fs.lstatSync(path).isDirectory();
+}
 
 function getFileNames(directiveName) {
   return [`./src/${directiveName}/${directiveName}.ts`];
 }
-console.log(JSON.stringify(doc(getFileNames('tabset'))));
+
+function fileExists(file) {
+  let result = false;
+  try {
+    fs.accessSync(file);
+    result = true;
+  }
+  catch (e) {
+    console.error(`the specified file does not exist: ${file}`)
+  }
+
+  return result;
+}
+
+function getComponentMap() {
```

Comment:
@icfantv I think that we should simply use an existing globbing library, ex.: https://www.npmjs.com/package/glob